### PR TITLE
Improve CLI UI

### PR DESCRIPTION
**Pull request by AjayShanker-geek**
- Add Messages Java
- Add Colors
- Update Ui display

### CODE REVIEW

**Review by NCF3535 — APPROVED** [review]
LGTM

File: src/main/java/seedu/duke/commands/BackCommand.java [hunk 1]
```diff
@@ -29,11 +29,11 @@ public CommandResult execute() {
 
         if (state.getState() == StateType.TASK_STATE) {
             state.setState(StateType.MAIN_STATE);
-            System.out.println(MESSAGE_SWITCHED_TO_MAIN);
+            // System.out.println(MESSAGE_SWITCHED_TO_MAIN);
             return new CommandResult(MESSAGE_SWITCHED_TO_MAIN);
         } else {
             logger.log(Level.WARNING, "Attempted to switch back, but already in MAIN_STATE.");
```
Comment (NCF3535) [inline src/main/java/seedu/duke/commands/BackCommand.java:35]: Maybe we couls catergorise the logger as INFO instead of WARNING

File: src/main/java/seedu/duke/common/Messages.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+               /         \\
+               |`-.....-'|
+               | MediTask|
+             _ |`-.....-'|     _
+            (\\)`-.__.__.(I) _(/)
+              (I)  (/)(I)(\\)
+                 (I)        Task management for medical professionals
+                        """;;
+
+    // Messages
```
Comment (NCF3535) [inline src/main/java/seedu/duke/common/Messages.java:24]: Maybe we can further group them to prompts and welcome/goodbye so it is easier to manage down the line as we have more messages.

File: src/main/java/seedu/duke/ui/Colors.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package seedu.duke.ui;
+
+/**
+ * Color class contains ANSI color codes for text color in the console.
+ */
+public class Colors {
```
Comment (NCF3535) [inline src/main/java/seedu/duke/ui/Colors.java:6]: Maybe we could add a method to handle colored text. Example:  public static String colorize(String text, String colorCode) {
        return colorCode + text + ANSI_RESET;
    }

File: src/main/java/seedu/duke/ui/Ui.java [hunk 4]
```diff
@@ -17,14 +19,13 @@ public Ui() {
 
     public void showMainScreen() {
         showLine();
-        System.out.print("Meditask: ");
+        System.out.print(Messages.MESSAGE_MAIN_PROMPT);
     }
 
     public void showTaskScreen(String patientName) {
         showLine();
         System.out.println("Patient: " + patientName);
```
Comment (NCF3535) [inline src/main/java/seedu/duke/ui/Ui.java:27]: Perhaps we could colourise this to make it more disctinct.

File: src/test/java/seedu/duke/ui/UiTest.java [hunk 5]
```diff
@@ -73,7 +89,6 @@ public void showWelcome_displaysWelcomeMessage_correctly() {
     @Test
     public void closeScanner_noExceptions_closesCorrectly() {
```
Comment (NCF3535) [inline src/test/java/seedu/duke/ui/UiTest.java:90]: Could add more tests for edge cases where it does throw an exception.

**Review by NCF3535 — APPROVED** [review]

**Review by yuanners99 — COMMENTED** [review]
Good use of modularity, keeping the functions focused and manageable. Remember to remove any commented-out code and add more comments to enhance readability.
